### ISSUE TITLE
Pyright doesn't work on line break due to line length

### ISSUE THREAD

**Issue by Paulchen5** [microsoft/pyright#10556]
**Describe the bug**
The following python code gets highlighted by Pylance in VS Code with the report of private usage
```py
query_plan = utils.__get_original_query_plan(
            file_content
        )  # pylint: disable=protected-access # pyright: ignore[reportPrivateUsage]
```

This code doesn't get highlighted:
```py
query_plan = utils.__get_original_query_plan(file_content)  # pylint: disable=protected-access # pyright: ignore[reportPrivateUsage]
```

**Code or Screenshots**
code example above: ⬆ 

<img width="563" alt="Image" src="https://github.com/user-attachments/assets/4334e366-dcca-43b1-9910-f3d89dfe447e" />

<img width="784" alt="Image" src="https://github.com/user-attachments/assets/d576d2b7-00bb-4e61-be27-1050d4d988b0" />

**VS Code extension or command-line**
VS Code extension `[pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)` which seems to use pyright; pylance version: `2025.5.1`

**Comment by erictraut** [comment]
Yes, that's correct. The `# pyright: ignore` and `# type: ignore` suppress diagnostics only on the line where they appear. If the diagnostic spans multiple lines, the `# pyright: ignore` or `# type: ignore` will work when applied to any of those lines. In your example, the diagnostic is associated with the range `utils.__get_original_query_plan`, so if you want to suppress it with a `# pyright: ignore`, you need to ensure that the comment is on that line.

```python
query_plan = utils.__get_original_query_plan(  # pyright: ignore[reportPrivateUsage]
    file_content
)
```
